### ISSUE TITLE
arm64 build

### ISSUE THREAD

**Issue by xgdgsc** [xanderfrangos/twinkle-tray#506]
Hope support native arm64 build: https://www.electronjs.org/docs/latest/tutorial/windows-arm/

**Comment by xanderfrangos** [comment]
I don't have an actual ARM device to test with, but does this build work?

https://1drv.ms/u/s!AiWKFpjcSKRQhtxuXoFk_piyxOTn5A?e=eemA7e

**Comment by xgdgsc** [comment]
Thanks. It works.

**Comment by xanderfrangos** [comment]
Great, thanks for confirming! I don't have any plans to include ARM64 builds with each update right now, but I have set up a script to compile it. If the Microsoft Store will allow me to upload both x64 and ARM64, I may do that.

Does the regular x64 version work as-is?

**Comment by xgdgsc** [comment]
Yes, x64 version a bit more sluggish on start. Some apps in Store provide ARM64 builds.
